BazyDanych/BazaDanych/BazaDanych/DbConnection/Db.cs

```diff
@@ -0,0 +1,124 @@
+﻿using BazaDanych.Model;
+using Dapper;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace BazaDanych.DbConnection
+{
+    internal class Db
+    {
+        private readonly string _connectionString;
+
+        public Db(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+
+        public int AddAuthor(Author author)
+        {
+            using var connection = Connect();
+            connection.Open();
+            var procedure = "[uspAddAuthor]";
+            var values = new { Name = author.Name, Surname = author.Surname };
+            var result = connection.Execute(procedure, values, commandType: CommandType.StoredProcedure);
+            connection.Close();
+            return result;
```

Comment:
Q: Co to oznacza że zwracasz `int`? Moim zdaniem tutaj powineneś mieć jakiegoś ifa który Ci sprawdzi rezultat i ewentualnie wyrzuci wyjątek. W nowym programówaniu nie zwraca się statusu wykonania metody zazwyczaj.

Reply:
[8b39498](https://github.com/broniuu/Praktyki2022/pull/3/commits/8b39498b8db6dbab70fd59889720f31b39bf7630)

Reply:
[8b39498](https://github.com/broniuu/Praktyki2022/pull/3/commits/8b39498b8db6dbab70fd59889720f31b39bf7630)